Dockerfile

```diff
@@ -6,10 +6,16 @@ RUN apt-get update && apt-get install -y \
     libssl-dev \
     logrotate \
     sudo \
+    curl \
+    jq
 && rm -rf /var/lib/apt/lists/*
 
 ENV VERNEMQ_VERSION 1.2.3
 
+# Defaults
+ENV DOCKER_VERNEMQ_KUBERNETES_NAMESPACE default
```

Comment:
Defining those environment variables in the Dockerfile using `ENV` would also set them in the resulting Docker container. This isn't what we want in a non-Kubernetes case. 
I guess the Kubernetes autodiscovery would still work after removing them from the Dockerfile.

Reply:
Uhm, that's true. They shouldn't affect anything in case kubernetes autodiscovery isn't enabled, they're just there for providing defaults. In case you don't like this, I can remove the ENV from Dockerfile and manage the defaults case in the bash script